spec/controllers/api/v1/organizations_controller_spec.rb

```diff
@@ -146,5 +146,27 @@
         it_behaves_like "supports update_links"
       end
     end
+
+    describe "include avatar and background" do
```

Comment:
can we test this in the serializer spec instead of the controllers? I've been working on extracting them recently when i see them.